### PR TITLE
feat: bulletin board ruby gem

### PR DESCRIPTION
**Pull request by slickepinne**
This PR adds a new bulletin board ruby gem to be used by Decidim and the Bulletin Board app.

### CODE REVIEW

File: decidim-bulletin_board-ruby/spec/decidim/bulletin_board_spec.rb [hunk 1]
```diff
@@ -1,11 +1,7 @@
 # frozen_string_literal: true
```
Comment (beagleknight) [inline decidim-bulletin_board-ruby/spec/decidim/bulletin_board_spec.rb:1]: We can remove this test now 😄

File: decidim-bulletin_board-ruby/spec/fixtures/bb_schema.json [hunk 2]
```diff
@@ -0,0 +1,1806 @@
+{
```
Comment (beagleknight) [inline decidim-bulletin_board-ruby/spec/fixtures/bb_schema.json:1]: I'm not sure if we need to check out this file in the repository yet 🤔  Let's leave it for now, but we should think about that in the future

**Review by beagleknight — APPROVED** [review]
Awesome ✨

**Review by agustibr — APPROVED** [review]
👏

**Pull request merged**